.github/workflows/continuous_integration.yml

```diff
@@ -39,7 +39,7 @@ jobs:
             --package=com.azavea.ci \
             --default-api-port=8080 \
             --default-db-port=5432
-          cd ci && sbt ";bloopInstall; scalafmt; scalafmtSbt"
+          cd ci && sbt ";bloopInstall; scalafmt; scalafmtSbt; scalafix --check"
```

Comment:
This will catch any future `scalafix` lint errors introduced into the scaffolded project.